modules/gapi/src/compiler/gcompiler.cpp

```diff
@@ -176,6 +176,103 @@ namespace
     }
 } // anonymous namespace
 
+namespace validation
+{
+    class meta_matcher_visitor : cv::util::static_visitor<bool, meta_matcher_visitor>
```

Comment:
Normally, namespaces in OpenCV don't increase the indent. This file is pretty old and we were not aware of that requirement back then - but I'd recommend to follow the guideline for the new code we develop

Reply:
ok, i'll align it